wezterm-input-types/src/lib.rs

```diff
@@ -516,6 +518,10 @@ impl TryFrom<String> for Modifiers {
                 mods |= Modifiers::CTRL;
             } else if ele == "SUPER" || ele == "CMD" || ele == "WIN" {
                 mods |= Modifiers::SUPER;
+            } else if ele == "META" {
```

Comment:
Defining `META` as its own thing needs some care and attention throughout the wezterm crates as well as the termwiz crate, as we've always treated it as an alias of `ALT`, which you can see happening on line 515 above.

This particular line in this PR has no effect because it is evaluated after that line.

Breaking it out into its own thing needs to consider:
* What happens for non-kitty encodings when this modifier is used? Is it simply ignored?
* How does it get set? This PR doesn't seem to add any OS-specific logic to map the incoming key press from the windowing layer to actually set either the META or HYPER flags
* What happens to folks that have defined key assignments using META as the modifier? Do those effectively stop working when they upgrade? Is that ok? (probably?)